<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>644</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>818</width>
    <height>644</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SCN</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{

background:transparent;
border-radius:20px;


};

</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#frame{

border: 2px solid  rgb(0, 255, 200);

}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">QFrame{
border-radius:20px;
background-color:  rgba(0, 0, 0, 0.7);


};


</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QFrame" name="titleBar">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);

border: 2px solid  rgb(0, 255, 200);



</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="APPNAME">
            <property name="styleSheet">
             <string notr="true">color:rgb(255,255,255);
background:transparent;
font: 87 20pt &quot;Arial Black&quot;;
border: 1px solid rgd(0,0,0);
border: 1 solid rgb(0,0,0)</string>
            </property>
            <property name="text">
             <string>SCN</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="minimiz">
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-color: rgb(0, 255, 255);
height:20px;
border-radius:10px;
border: 1 solid rgb(0,0,0)
}

QPushButton:hover{
background-color: rgb(0, 129, 129);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Exit">
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
background-color: rgb(255, 0, 0);
height:20px;
border-radius:10px;
border: 1 solid rgb(0,0,0)

}

QPushButton:hover{
background-color: rgb(144, 0, 0);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QScrollArea" name="scrollArea">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>429</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background:transparent;</string>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>776</width>
              <height>403</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">border-color: rgb(255, 0, 0);</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item alignment="Qt::AlignHCenter">
              <widget class="QLabel" name="qimage">
               <property name="maximumSize">
                <size>
                 <width>800</width>
                 <height>500</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="layout3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QComboBox" name="WindowList">
               <property name="styleSheet">
                <string notr="true">
padding: 2 20;
background:transparent;
color: rgb(255, 255, 255);
border:1px solid  rgb(0, 255, 255);
border-radius:10px;
height: 20px;
</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="resolutions">
               <property name="minimumSize">
                <size>
                 <width>50</width>
                 <height>0</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">padding: 2 20;
background:transparent;
color: rgb(255, 255, 255);
border:1px solid  rgb(0, 255, 255);
border-radius:10px;
height: 20px;
</string>
               </property>
               <item>
                <property name="text">
                 <string>HD</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>FHD</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="bitRateCombo">
               <property name="styleSheet">
                <string notr="true">padding: 2 20;
background:transparent;
color: rgb(255, 255, 255);
border:1px solid  rgb(0, 255, 255);
border-radius:10px;
height: 20px;
</string>
               </property>
               <item>
                <property name="text">
                 <string>200kbps</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>300kbps</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>400kbps</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>500kbps</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="RefreshBtn">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">

QPushButton{
color: rgb(255, 255, 255);
border:1px solid  rgb(0, 255, 255);
border-radius:10px;
height:30px;
}

QPushButton:hover{
	background-color: rgb(0, 125, 125);
}</string>
               </property>
               <property name="text">
                <string>Refresh</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="RecordBtn">
               <property name="styleSheet">
                <string notr="true">
QPushButton{
color: rgb(255, 255, 255);
border:1px solid  rgb(0, 255, 255);
border-radius:10px;
height:30px;
}
QPushButton:hover{
background-color: rgb(100, 0, 0);
}</string>
               </property>
               <property name="text">
                <string>Record</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="PauseBtn">
               <property name="styleSheet">
                <string notr="true">QPushButton{
color: rgb(255, 255, 255);
border:1px solid  rgb(0, 255, 255);
border-radius:10px;
height:30px;
}

QPushButton:hover{
background-color: rgb(148, 148, 148);
}</string>
               </property>
               <property name="text">
                <string>Pause</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="PreviewCheck">
               <property name="styleSheet">
                <string notr="true">padding: 2 20;
height:20;
border-radius:10px;


color: rgb(255, 255, 255);
border:1px solid  rgb(0, 255, 255);</string>
               </property>
               <property name="text">
                <string>Preview</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item alignment="Qt::AlignHCenter">
            <widget class="QRadioButton" name="radioButton1">
             <property name="styleSheet">
              <string notr="true">padding: 2 20;
height:20;
border-radius:10px;


color: rgb(255, 255, 255);
border:1px solid  rgb(0, 255, 255);
</string>
             </property>
             <property name="text">
              <string>Object detection</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QPushButton" name="AimBotBtn">
               <property name="styleSheet">
                <string notr="true">

QPushButton{
color: rgb(255, 255, 255);
border:1px solid  rgb(0, 255, 255);
border-radius:10px;
height:30px;
}</string>
               </property>
               <property name="text">
                <string>AimBot</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="styleSheet">
                <string notr="true">
QPushButton{
color: rgb(255, 255, 255);
border:1px solid  rgb(0, 255, 255);
border-radius:10px;
height:30px;
}</string>
               </property>
               <property name="text">
                <string>PushButton</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="lableNOTIFY">
         <property name="styleSheet">
          <string notr="true">font: 87 12pt &quot;Arial Black&quot;;
color: rgb(255, 0, 0);
background:transparent;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item alignment="Qt::AlignLeft">
          <widget class="QFrame" name="resizegrip2">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background: transparent;</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="credit">
           <property name="styleSheet">
            <string notr="true">background: transparent;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignRight">
          <widget class="QFrame" name="resizegrip">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background:transparent;</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="1.qrc"/>
 </resources>
 <connections/>
</ui>
